pkg/p2p/libp2p/internal/handshake/handshake_test.go

```diff
@@ -140,9 +140,8 @@ func TestHandshake(t *testing.T) {
 	t.Run("ERROR - ack write error ", func(t *testing.T) {
 		testErr := errors.New("test error")
 		expectedErr := fmt.Errorf("ack: write message: %w", testErr)
-
 		expectedInfo := Info{
-			Address:   "node2",
+			Address:   []byte("node2"),
```

Comment:
Use swarm.Address not []byte as type, and construct it from actual byteslice or use swarm.NewAddress if it is constructed from hex-encoded string.